templates/repmgr.conf.j2

```diff
@@ -0,0 +1,394 @@
+# {{ ansible_managed }}
```

Comment:
We should be able to handle different repmgr versions... as the config file in later versions has new parameters, which will break earlier versions.  It should work similarly to how the postgresql.conf files work for the main role.

Reply:
This has been adapted to work the same way as for postgresql.conf as requested.